css-pseudo-4/Overview.bs

```diff
@@ -1337,6 +1337,43 @@ details[open]::details-content {
 }</pre>
   </div>
 
+<h3 id="picker-pseudo">
+Styling form control pickers: the ''::picker()'' pseudo-element</h3>
+
+  The <dfn>::picker()</dfn> pseudo-element targets the popup picker of form
+  control elements which have popup pickers, such as the <{select}> element. It
+  is a [=part-like pseudo-element=].
+
+  <pre class=prod>
+    ::picker() == ::picker( <<ident>>+ )
+  </pre>
+
+  The ''::picker()'' pseudo-element only matches when the <a>originating
+  element</a> supports <a>base appearance</a> and has a popup picker. The
+  specified <<ident>> must also match the unique picker name of the
+  <a>originating element</a>. For example, the unique picker name for the
+  <{select}> element is "select".
+
+  There is no restriction on which properties apply to the ''::picker()''
+  pseudo-element.
```

Comment:
This is implied by it being part-like, although maybe it's worth stating anyway.  But maybe start with "Since it is a [=part-like pseudo-element=], ".

Reply:
I copied this from a bunch of other part-like pseudo-elements in this spec like `::file-selector-button` and `::details-content`.

> But maybe start with "Since it is a [=part-like pseudo-element=], ".

Done

Reply:
Yeah, a bunch of stuff needs cleaning up, but we may as well set a good example here.